### PR TITLE
DB: Deprecate - copy_elementor_meta (ED-2088)

### PR DESCRIPTION
**Pull request by Nevoss**
~Depends on: https://github.com/elementor/elementor/pull/12897~
~Depends on: https://github.com/elementor/elementor/pull/12879~

### CODE REVIEW

**Comment by matipojo** [comment]
We will ope

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull 3.3.0 and fix the CI errors.

File: includes/db.php [hunk 1]
```diff
@@ -283,26 +283,29 @@ public function iterate_data( $data_container, $callback, $args = [] ) {
...
 	 */
 	public function safe_copy_elementor_meta( $from_post_id, $to_post_id ) {
-		// It's from  WP-Admin & not from Elementor.
-		if ( ! did_action( 'elementor/db/before_save' ) ) {
+		Plugin::$instance->modules_manager
+			->get_modules( 'dev-tools' )
+			->deprecation
+			->deprecated_function(
+				__METHOD__,
+				'3.1.0',
```
Comment (rodolphebertozzo) [inline includes/db.php:298]:
```suggestion
				'3.3.0',
```

File: includes/db.php [hunk 2]
```diff
@@ -313,34 +316,29 @@ public function safe_copy_elementor_meta( $from_post_id, $to_post_id ) {
...
-		foreach ( $from_post_meta as $meta_key => $values ) {
-			// Copy only meta with the `_elementor` prefix
-			if ( 0 === strpos( $meta_key, '_elementor' ) || in_array( $meta_key, $core_meta, true ) ) {
-				$value = $values[0];
+		Plugin::$instance->modules_manager
+			->get_modules( 'dev-tools' )
+			->deprecation
+			->deprecated_function(
+				__METHOD__,
+				'3.1.0',
```
Comment (rodolphebertozzo) [inline includes/db.php:331]:
```suggestion
				'3.3.0',
```

**Review by rodolphebertozzo — CHANGES_REQUESTED** [review]

File: includes/db.php (hunk 1 above)
Comment (rodolphebertozzo) [inline includes/db.php:298]:
```suggestion
				'3.4.0',
```

File: includes/db.php (hunk 2 above)
Comment (rodolphebertozzo) [inline includes/db.php:331]:
```suggestion
				'3.4.0',
```

**Review by rodolphebertozzo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**